bindings/csharp/sdk/README.md

```diff
@@ -18,13 +18,13 @@ This quickstart includes one service:
 name: Run and initialize PostgreSQL container
 expected_return_code:
 background: true
-sleep: 5
-timeout_seconds: 6
+sleep: 60
+timeout_seconds: 300
```

Comment:
**Background to the problem:**

As we know docker installation in macos isn't stable in github runner, makes the workflow flaky. Due to which we are using podman. And "docker compose up" doesn't work with podman, so I had to create a symbolic link between docker-compose and podman-compose. With this change workflow was failing as it used to times out.

So, to overcome the timeout issue, I increased the timeout. I think I can reduce it a little and test again. but it will probably will be little higher than usual due to podman-compose being little slow (I guess).